sunpy/io/tests/test_asdf.py

```diff
@@ -0,0 +1,20 @@
+from sunpy.data.test import get_test_filepath
+from sunpy.io._asdf import get_header, read, write
+from sunpy.io.header import FileHeader
+
+AIA_MAP = get_test_filepath("aiamap_genericmap_1.0.0.asdf")
+
+
+def test_read():
+    cont = read(AIA_MAP)
+    assert isinstance(cont,list)
```

Comment:
it'll be better to have more checks here since list types still leave a lot of room for things to go wrong. some starters could be checking for the length or particular values but having some checks for Map-specific behaviour would be required eventually. I'd recommend taking a look at other tests in the directory to get a better idea of what could be useful.
the other two tests would benefit from a similarly expansion as well!